bin/render-pdf.js

```diff
@@ -12,6 +12,8 @@ const puppeteer = require("puppeteer");
 
 const inputPath = `file://${process.argv[2]}`;
 const outputPath = process.argv[3];
+const basic_auth_username = process.argv[4];
+const basic_auth_password = process.argv[5];
```

Comment:
There's no way to turn these into symbols is there?It would be nice to have something like `process[:username]`